src/QMCWaveFunctions/BsplineFactory/SplineC2ROMP.h

```diff
@@ -263,7 +263,7 @@ struct SplineC2ROMP : public SplineAdoptorBase<ST, 3>
   ///GGt data pointer
   const ST* master_GGt_ptr;
 
-  SplineC2ROMP() : BaseType(), nComplexBands(0), SplineInst(nullptr), MultiSpline(nullptr)
+  SplineC2ROMP() : Base(), nComplexBands(0), SplineInst(nullptr), MultiSpline(nullptr)
```

Comment:
Did you overlook changing this to
```C++
SplineC2ROMP() : Base(), nComplexBands(0)
```

Reply:
It also look like to me if the object is constructed through the copy constructor MultiSpline will never be valid.  If it is valid you'll call delete on a std::shared_ptr which you should not.

Reply:
Not done yet. I was waiting for the other PR touching this class. Cleaning it now.

Reply:
Done.